vdev/src/commands/package/msi.rs

```diff
@@ -1,58 +1,83 @@
 use anyhow::Result;
 
+
+
 #[cfg(windows)]
 use {
     crate::{app, util},
+    crate::app::CommandExt,
+    std::ffi::{OsStr},
     std::env,
     std::fs,
     std::iter::once,
-    std::path::Path,
+    std::path::{Path, PathBuf},
+    std::process::Command,
 };
 
+
+// Use the `bash` interpreter included as part of the standard `git` install for our default shell
+// if nothing is specified in the environment.
+#[cfg(windows)]
+const DEFAULT_SHELL: &str = "C:\\Program Files\\Git\\bin\\bash.EXE";
 /// Create a .msi package for Windows
 #[derive(clap::Args, Debug)]
 #[command()]
 pub struct Cli {}
 
 impl Cli {
     pub fn exec(self) -> Result<()> {
+        #[cfg(not(windows))]
+        {
+            println!("Sorry, the package-msi command is not supported on non-Windows platforms. Nothing was performed.");
+        }
         #[cfg(windows)]
         {
             // TODO: wait for other PR to be merged then replace the below line with app::version()
```

Comment:
Is this todo still waiting on another PR? If so, can we include the PR number?

Reply:
This seems to be removed on the most recent commit I don't see this TODO anymore on my local branch nor the files diff on github